disorder-fsm/src/Disorder/FSM.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
GeneralizedNewtypeDeriving is safe and should be used with a smile on your face. At least I do.